backend-api/template.yaml

```diff
@@ -18,6 +18,18 @@ Mappings:
     production: 
       ApiBurstLimit: 0
       ApiRateLimit: 0
+  TxMA:
+    dev:
+      TxmaAccount: arn:aws:iam::248098332657:root
+    build:
+      TxmaAccount: arn:aws:iam::750703655225:root
+    staging:
+      TxmaAccount: arn:aws:iam::178023842775:root
+    integration:
+      TxmaAccount: arn:aws:iam::729485541398:root
+    production:
+      TxmaAccount: arn:aws:iam::451773080033:root
+
 Parameters:
   Environment:
```

Comment:
Can we get some `AllowedValues` on the environment to limit it to our environment names. especially if they're being used to pull out variables from the mappings.

Reply:
added